recipes/libcamera-rpi/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set version = "0.4.0+rpt20250213" %}
+{% set url_safe_version = version | urlencode %}
+
+package:
+  name: libcamera-rpi
+  version: {{ version }}
+
+source:
+  url: https://github.com/raspberrypi/libcamera/releases/download/v{{ url_safe_version }}/libcamera-{{ url_safe_version }}.tar.xz
```

Comment:
is this a patched package specifically for the raspeberry pi?

this isn't the "conda-forge way" and we would typically just include the patches in the `arm` build and call it a day. 

what do you think? are the patches major?

the problem is that the symbols and python module name might crash.

Reply:
The only thing I know is that the raspberry pi fork is necessary for the picamera2 library, which is not guaranteed to work with the standard libcamera.

And judging from [this](https://github.com/raspberrypi/libcamera/compare/v0.4.0...v0.4.0+rpt20250127) the differences are quite large...

Reply:
but yes I get your point about the symbols/names clash, not sure how to handle this properly... Is there a way to express mutual exclusivity in conda-forge? Some way to say "package A and package B cannot be be installed together".

Reply:
There are ways to handle exclusive packages, but the easiest by far is to have "different variants".

If things are ABI compatible (not sure if they are) then that is by far the easiest thing to do.

we don't typically pull different sources for variants, but maybe this is an exceptional case that warrants this.

we could add "rpi" to the build string or something.... i typically don't like that but you know....


The easiest way to make a mutex is always putting it all in the same recipe.

Reply:
xref: https://github.com/raspberrypi/libcamera/issues/251

Reply:
ok, the two forks are ABI compatible, we can try the build variants with different sources...

I will try to search for some feedstock where this is already implemented to use as a reference.